Update requirements and docs migration (#11170)
open-metadata/OpenMetadata: openmetadata-docs-v1/content/v1.0.0/connectors/ingestion/workflows/profiler/auto_tagging.md

---

## Auto PII Tagging

> Here, we are tagging PII Sensitive/NonSensitive tag to column based on the following ways.

Reviewer: ```suggestion
Auto PII tagging for Sensitive/NonSensitive at the column level is performed based on the two approaches described below.
```

> PII Tagging is only available during `Profiler Ingestion`.

Reviewer: ```suggestion
{% note %}
PII Tagging is only available during `Profiler Ingestion`.
{% /note %}
```

### During Profiler Ingestion

> - During profiler ingestion, we profile sample data.

Reviewer: ```suggestion
- If sample data ingestion is enabled, during the profiler workflow OpenMetadata will infer the status of a column (PII or not) based on its content.
```

> - First we pass the column name through a regex, which identify credit card,email , etc.

Reviewer 2: ```suggestion
- First, we pass the column name through a regex, which identifies credit card, email, etc.
```

> - First we pass the column name through a regex, which identify credit card,email , etc.

Reviewer: Shouldn't this be before the bullet point above if it is "first"?

Author: So here `first` signifies the first method for scanning PII

Reviewer: Let's say we have sample data as well. Do we do 1. column scanning and 2. sample data scanning?

Author: Yes 👍

> - If it is not parse through regex, we use [presidio](https://microsoft.github.io/presidio/) library, which give spacy entity name and score, which is used to determine PII Sensitive/NonSensitive Tag.

Reviewer 2: ```suggestion
- If it is not parsed through the regex, we use [presidio](https://microsoft.github.io/presidio/) library, which gives spacy entity name and score, which is used to determine PII Sensitive/NonSensitive Tag.
```

> - If it is not parse through regex, we use [presidio](https://microsoft.github.io/presidio/) library, which give spacy entity name and score, which is used to determine PII Sensitive/NonSensitive Tag.

Reviewer: ```suggestion
- If the column status (PII or not) cannot be parsed with regex, we will use [presidio](https://microsoft.github.io/presidio/) library, which allows OpenMetadata to determine the PII status.
```

> - `confidence` parameter is passed determine minimun score required to tag the column.

Reviewer 2: ```suggestion
- `confidence` parameter is passed to determine the minimum score required to tag the column.
```
